tracer/tracer.go

```diff
@@ -153,6 +153,8 @@ type Config struct {
 	FilterErrorFrames bool
 	// KernelVersionCheck indicates whether the kernel version should be checked.
 	KernelVersionCheck bool
+	// VerboseMode indicates whether to load the debug version of tracer.
+	VerboseMode bool
```

Comment:
We should be more specific here, and not propagate the cmdline argument naming.

```suggestion
	// DebugTracer indicates whether to load the debug version of eBPF tracers.
	DebugTracer bool
```